src/server/routes/index.ts

```diff
@@ -226,6 +227,7 @@ export const withRoutes = async (fastify: FastifyInstance) => {
   await fastify.register(sendSignedUserOp);
   await fastify.register(getTxHashReceipt);
   await fastify.register(getUserOpReceipt);
+  await fastify.register(getTxLogs);
```

Comment:
Nit: I'd like to avoid `tx` in code even if internal facing.

Reason: I'd like to remove `tx` in external facing code to match the thirdweb SDK, and it's just simpler to get in the habit of spelling the whole thing out.